src/app/app.component.html

```diff
@@ -23,5 +23,7 @@ <h2>Settings</h2>
     <h1 class="title">Hello, World!</h1>
     <adv-code-line code="test" title="GPG Key" />
   </section>
-  <adv-alert content="Hello, World!" [actions]=testAlert>Alert!</adv-alert>
+  <adv-alert title="Test Alert" [hidden]=false content="Hello, World!" [actions]=alertActions>Alert!</adv-alert>
+  <button (click)=openActionSheet()>ActionSheet</button>
+  <adv-action-sheet [actions]=actionSheetActions></adv-action-sheet>
```

Comment:
I always find it quite irritating that you don't add any quotation marks around property bindings or methods. It makes the property or event look like a normal string in my code editor due to the syntax highlighting. 

```diff
- <button (click)=openActionSheet()>ActionSheet</button>
+ <button (click)="openActionSheet()">ActionSheet</button>
```